tests/test_write_prescriptions.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+from uuid import uuid4
+
+import pandas as pd
+import pytest
+from ata_db_models.models import Prescription
+from sqlalchemy.engine import Engine
+from sqlalchemy.orm import sessionmaker
+from sqlmodel import select
+
+from ata_pipeline1.helpers.enums import SiteName
+from ata_pipeline1.write_prescriptions import write_prescriptions
+from tests._helpers import create_and_drop_tables
+
+
+@pytest.fixture
+def prescriptions() -> list[Prescription]:
+    return [
+        Prescription(user_id=uuid4(), site_name=SiteName.AFRO_LA, prescribe=True, last_updated=datetime.now())
+        for _ in range(3)
+    ]
+
+
+@pytest.fixture
+def prescriptions_as_df(prescriptions: list[Prescription]) -> pd.DataFrame:
+    return pd.DataFrame(data=[p.dict() for p in prescriptions])
+
+
+def test_write_prescriptions(prescriptions_as_df: pd.DataFrame, engine: Engine, session_factory: sessionmaker) -> None:
+    with create_and_drop_tables(engine):
+        rows_written = write_prescriptions(df=prescriptions_as_df, session_factory=session_factory)
+        assert rows_written == prescriptions_as_df.shape[0]
+        with session_factory.begin() as session:  # type: ignore
+            statement = select(Prescription)
+            results = session.execute(statement)
+            data = [prescription[0].dict() for prescription in results]
+    print(prescriptions_as_df)
```

Comment:
⛏ Assuming you want to delete this before merging.

Reply:
hah yup ty